### DISCUSSION TITLE
VEX outputs support question

### DISCUSSION THREAD

**Discussion by jni2000** [e-m-b-a/emba#1399] [Q&A]
Besides the great and nice html output, does EMBA generate any popular VEX outputs like 
1. CSAF: (https://oasis-open.github.io/csaf-documentation/), 
2. CycloneDX: (https://cyclonedx.org/capabilities/vex/) and 
3. OpenVEX: (https://edu.chainguard.dev/open-source/sbom/what-is-openvex/)

**Comment by m-1-k-3** [comment]
Currently EMBA only supports Cyclonedx SBOMs. VEX is on the roadmap but we need some refactoring of F20 first.

**Comment by jni2000** [comment]
Thanks, @m-1-k-3!
